src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ContextHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.maui;
+
+import android.content.res.TypedArray;
+import android.content.Context;
+import java.lang.reflect.Field;
+
+public class ContextHelper {
```

Comment:
Can you put the new method here in `ViewHelper.java` instead? JonP was saying we pay startup cost per Java type we call from C#, so we could have a giant single JavaType to avoid that... 🤦‍♂️

Reply:
lol, that was where I put it first. But I moved it becuse it wasn't a ViewHelper. 
If we do add it all to one type then we should probably rename ViewHelper to something more generic.

Reply:
I moved this code to `PlatformHelper`